frontend/src/components/MapView/DateSelector/index.tsx

```diff
@@ -93,6 +93,16 @@ function DateSelector({
   const [pointerPosition, setPointerPosition] = useState<Point>({ x: 0, y: 0 });
 
   const dateRef = useRef(availableDates);
+  const maxDate = new Date(Math.max(...availableDates, new Date().getTime()));
+
+  // eslint-disable-next-line no-console
+  console.log('Available dates:');
+  // eslint-disable-next-line no-console
+  console.log(availableDates.map(n => new Date(n)));
+  // eslint-disable-next-line no-console
+  console.log('Max date:');
+  // eslint-disable-next-line no-console
+  console.log(maxDate);
```

Comment:
@ericboucher . Are those console.logs necessary? I am assuming that are for testing purposes.